lib/rack/error_handler.rb

```diff
@@ -0,0 +1,23 @@
+module Rack
+  class ErrorHandler
+    def initialize(app)
+      @app = app
+    end
+
+    def bad_request(error)
+      [400, {}, ["Bad Request: #{error.message}"]]
```

Comment:
This could potentially leak sensitive information.  We should not include ruby exception messages in response bodies.  Same for `internal_server_error`.

Reply:
What about the error class name?